_includes/contact_form.html

```diff
@@ -410,6 +410,7 @@ <h3>{{ site.data.[page.lang].settings.contact_page.support_request_form.title }}
     method="POST"
     onSubmit="return remapvalues(this); return true;"
```

Comment:
Wonder if this code might be conflicting with the `preventDefault`.

Reply:
should we hook the `verifyCanSubmitEntry` into that implementation?

Reply:
I'd probably go the other way around. Worth testing locally that removing this line would make a difference. If it does, maybe we can move the `remapvalues` logic into the submit handler where we're doing the spam validation.

Depending on time urgency, specific implementation approach is not a blocker if it's easier to put the new code there.